### PR TITLE
create new profile rinshin jalal

### PR DESCRIPTION
**Pull request by Rinshin-Jalal**
Added a new profile!!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2718"></a>

### CODE REVIEW

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

Please fix your JSON profile naming. It should be `{your GitHub username}.json` -> `
Rinshin-Jalal.json`

If you need any help feel free to ask.

File: data/rinshin.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**A human-level Programmer | Who wants to learn more!!",
+  "tags": ["Open Source", "Nextjs", "Linux", "Django", "Node.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/just_me_rinz" },
+    { "icon": "FaGithub", "url": "https://github.com/Rinshin-Jalal" },
+    { "icon": "FaLink", "url": "https://rinshin.me" }
+  ],
+  "avatar": "https://avatars.githubusercontent.com/u/88965873?v=4",
```
Comment (senali-d) [inline data/rinshin.json:12]: Use short image link.
```suggestion
  "avatar": "https://github.com/Rinshin-Jalal.png",
```

**Review by senali-d — CHANGES_REQUESTED** [review]
@Rinshin-Jalal Revert the changes in `package.json` and `package.lock.json` files

**Comment by eddiejaoude** [comment]
It looks like you have used `npm install` instead of `npm ci` from the docs, this causes the generated files to be updated but these should not be committed without specific changes to them.
https://linkfree.eddiehub.io/docs/environments/local-development

Please remove these changes with:

- `git checkout main -- package.json`
- `git checkout main -- package-lock.json`

It is always best to check file changed with `git status -s` before doing a commit and never use `.` in your git commands as this will include files you may not want.

Any questions come and ask in the EddieHub Discord http://discord.eddiehub.org

**Comment by EmmaDawsonDev** [comment]
Do you need any help with the requested changes @Rinshin-Jalal ?

**Comment by eddiejaoude** [comment]
I will fix in a temporary branch

**Pull request merged**